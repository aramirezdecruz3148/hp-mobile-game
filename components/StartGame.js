import React from "react";
import { View, Button } from "react-native";
import PropTypes from "prop-types";

export default function StartGame({ setGame }) {
  return (
    <View>
      <Button 
        title="Start Game" 
        onPress={setGame}
      />
    </View>
  );
}

StartGame.propTypes = {
  setGame: PropTypes.func.isRequired
}