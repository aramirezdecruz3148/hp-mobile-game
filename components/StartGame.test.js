import React from 'react';
import { shallow } from 'enzyme';
import StartGame from './StartGame';

describe('StartGame component', () => {
  it('renders StartGame', () => {
    const wrapper = shallow(<StartGame 
      setGame={() => { }}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});