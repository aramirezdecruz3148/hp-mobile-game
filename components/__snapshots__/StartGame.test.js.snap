// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartGame component renders StartGame 1`] = `
<View>
  <Button
    onPress={[Function]}
    title="Start Game"
  />
</View>
`;
